index.js

```diff
@@ -122,7 +122,22 @@ function parse (opts, cb) {
       file = resolve.sync(filename, { basedir: path.dirname(file) })
     }
 
-    fs.readFile(file, 'utf8', read)
+    if (opts.transformer) {
+    	var cmd = opts.transformer.split(" ") ; 
+    	var args = cmd.slice(1).map(function(a){ return a=="$$"?file:a }) ;
+    	var p = require('child_process').spawn(cmd[0],args) ;
+    	var output = "" ;
+    	p.stdout.on('data',function(data){
+    		output += data.toString() ;
+    	}) ;
+    	p.on('close',function(){
+    		read(null,output) ;
+    	}) ;
+    	p.on('error',read) ;
+    	p.stderr.on('stderr',read) ;
```

Comment:
currently `read` will be called multiple times

- it will get called once for each buffer that comes out of `stderr`
- if `error` is emitted, it may also then then emit `close` when the stdio streams terminate, causing `read` to be called twice

make sure `read` only gets called once. it probably makes sense to not call `read` from inside the `.on('stderr'` but instead just print stderr out to the users terminal. then you only will call `read` from inside error or close